claim-check/code-samples/sample-2/FunctionConsumer2/FunctionConsumer2.csproj

```
@@ -0,0 +1,38 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+		<PropertyGroup>
+				<TargetFramework>net8.0</TargetFramework>
+				<AzureFunctionsVersion>v4</AzureFunctionsVersion>
+				<OutputType>Exe</OutputType>
+				<ImplicitUsings>enable</ImplicitUsings>
+				<Nullable>enable</Nullable>
+		</PropertyGroup>
+		
+		<ItemGroup>
+				<FrameworkReference Include="Microsoft.AspNetCore.App" />
+				<PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.21.0" />
+				<PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.17.2" />
+				<PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore" Version="1.2.1" />
+				<PackageReference Include="Microsoft.ApplicationInsights.WorkerService" Version="2.22.0" />
+				<PackageReference Include="Microsoft.Azure.Functions.Worker.ApplicationInsights" Version="1.2.0" />
+				<PackageReference Include="Azure.Identity" Version="1.11.0" />
```

While it's not the latest, it's what you used in the client.

```suggestion
				<PackageReference Include="Azure.Identity" Version="1.11.1" />
```